lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -45,13 +46,11 @@ class Expr
      *     // (u.type = ?1) AND (u.role = ?2)
      *     $expr->andX($expr->eq('u.type', ':1'), $expr->eq('u.role', ':2'));
      *
-     * @param Expr\Comparison|Expr\Func|Expr\Andx|Expr\Orx|string $x Optional clause. Defaults to null,
-     *                                                               but requires at least one defined
-     *                                                               when converting to string.
-     *
-     * @return Expr\Andx
+     * @param Expr\Comparison|Expr\Func|Expr\Andx|Expr\Orx|string ...$x Optional clause. Defaults to null,
+     *                                                                  but requires at least one defined
+     *                                                                  when converting to string.
      */
-    public function andX($x = null)
+    public function andX(...$x) : Expr\Andx
```

Comment:
This is a BC break for extending classes, and cannot go into 2.8 as such: https://3v4l.org/be7cf

Reply:
I have changed the target